apisix/plugins/opa.lua

```diff
@@ -125,6 +133,18 @@ function _M.access(conf, ctx)
         end
 
         return status_code, reason
+    else if result.headers and #conf.send_headers_upstream > 0 then
+            local headersToSend = {}
+            for key in pairs(conf.send_headers_upstream) do
+                headersToSend[key] = true
+            end
+           for key,value in pairs(result.headers) do
+                if headersToSend[key] then
+                    core.request.set_header(ctx,key,value)
+                end
+
+           end
+        end
```

Comment:
```suggestion
    else if result.headers and conf.send_headers_upstream then
            local headersToSend = {}
            for key in pairs(conf.send_headers_upstream) do
                headersToSend[key] = true
            end
           for key,value in pairs(result.headers) do
                if headersToSend[key] then
                    core.request.set_header(ctx,key,value)
                end

           end
        end
```

Reply:
The indent is not right, only four spaces are needed

Reply:
done